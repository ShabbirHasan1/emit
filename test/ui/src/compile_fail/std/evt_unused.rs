#![deny(unused_must_use)]

fn main() {
    emit::evt!("test");
}
