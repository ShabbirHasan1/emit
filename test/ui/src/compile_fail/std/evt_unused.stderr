error: unused return value of `__private_evt` that must be used
 --> src/compile_fail/std/evt_unused.rs:4:5
  |
4 |     emit::evt!("test");
  |     ^^^^^^^^^^^^^^^^^^
  |
  = note: this macro returns an `Event` without emitting it; send it through an `emit::Emitter`, or use the `emit::emit!` macro instead
note: the lint level is defined here
 --> src/compile_fail/std/evt_unused.rs:1:9
  |
1 | #![deny(unused_must_use)]
  |         ^^^^^^^^^^^^^^^
help: use `let _ = ...` to ignore the resulting value
  |
4 |     let _ = emit::evt!("test");
  |     +++++++
